.github/workflows/publish-sdk.yaml

```diff
@@ -0,0 +1,34 @@
+name: Publish Android SDK
+on:
+  release:
+    types: [published]
+
+env:
+  ANDROID_API_LEVEL: 33
+
+jobs:
+  lint-test-sdk:
+    runs-on: macos-latest
```

Comment:
Did you need to `macos` image to make this work?

Reply:
Agree this probably should not be needed